src/components/content-box/_content-box.scss

```diff
@@ -3,24 +3,39 @@ $include-html: false !default;
 @if ($include-html) {
 
   .mint-content-box {
+
+    &__title {
+      min-height: rhythm(2);
+    }
+
+    &__title ~ &__title {
+      margin-top: - rhythm(1/2);
+      margin-bottom: - rhythm(1/2);
+    }
+
     &__header {
-      vertical-align: middle;
-      margin-bottom: 3px;
+      @include components-container;
+      height: rhythm(2);
+    }
+
+    &__title ~ &__header {
```

Comment:
Not sure if it won't be separated afterwards. This 2 things are a bit different and may need to have different styles